### ISSUE TITLE
If any1 get this issue and my solution is working 4 u?

### ISSUE THREAD

**Issue by trinib** [trinib/AdGuard-WireGuard-Unbound-Cloudflare#2]
> > > I'm doing this via linux and get stuck at the "Now go to https://1.1.1.1/help" part, some times it'll pass, most times not but I'm using my router as the dhcp server
> > 
> > on pc or phone?, it's a browser thing.. and which browser you use?
> 
> Tried on pc via Firefox, Chrome on phone did pass but eventually failed, usually fails for DoH. Now I'm thinking it might be an FF issue

OK for Firefox go in dns settings and select custom and enter pi ipv4 address and then in proxy setting select auto detect for this network

for chrome set dns to automatic and for proxy if you are on windows it will take you windows proxy setting and turn off  automatically detect settings (should turn off this to if using Firefox as well I suppose)

then go in powershell and enter command:
      
      ipconfig /flushdns

Now restart pc and check it you get hit and miss with DoH. I had that issue at first and configured dns and proxy in browser and it was more stable.. only very rarely I get a miss which could be the site or browser.. let me know

_Originally posted by @trinib in https://github.com/trinib/AdGuard-WireGuard-Unbound-Cloudflare/issues/1#issuecomment-853293197_

**Comment by Rar9** [comment]
I´m having similar issues with getting Using DNS over HTTPS (DoH) = No

My Adguard  has this extra line as I`running a Unifi  USG
[/1.168.192.in-addr.arpa/]192.168.0.1

My unbound port is 5353 not 53 else it wont restart, as adguard is using it that to passes it to unifi to get all devices add free.

Any hit  to get this fixed?

**Comment by trinib** [comment]
> My unbound port is 5353 not 53 else it wont restart, as adguard is using it that to passes it to unifi to get all devices add free.
 
thats right .. i had tried different ports with unbound already but i cant remember if i had an issue or it made no difference to me .. if it works for you go ahead

> I´m having similar issues with getting Using DNS over HTTPS (DoH) = No

is it not showing at all ? or sometimes..

**Comment by Rar9** [comment]
Not showing at all.

DoT is sometimes not showing on first load.

**Comment by trinib** [comment]
hmm is DoH working without Unifi USG.. not sure.. you have to test to know exactly

**Comment by Rar9** [comment]
Not tested as it late, might have time on weekend
